app/views/Project/EntryEdit/index.tsx

```diff
@@ -380,8 +380,12 @@ function EntryEdit(props: Props) {
                     return;
                 }
                 const errors = entryBulk?.errors;
-                const deletedResult = response.project?.entryBulk?.deletedResult;
-                const saveResult = response.project?.entryBulk?.result;
+                const deletedResult = [
+                    ...response?.project?.entryBulk?.deletedResult ?? [],
+                ].reverse();
+                const saveResult = [
+                    ...response?.project?.entryBulk?.result ?? [],
+                ].reverse();
```

Comment:
We don't need to reverse anymore. If we do we can use the `reverse` prop on ListView, without modifying the data.